### PR TITLE
update `quick-xml` dependency to `0.28.1`

### LINKED ISSUE #26: Upgrade `quick-xml` dependency to 0.28.2
This requires a manual upgrade. See https://github.com/hannobraun/3mf-rs/pull/25.

### PR DESCRIPTION
**Pull request by JayceFayne**
Closes #26

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ repository = "https://github.com/hannobraun/3mf-rs"
 
 
 [dependencies]
-quick-xml = { version = "0.27.1", features = ["serialize"] }
+quick-xml = { version = "0.28.1", features = ["serialize"] }
```
Comment (hannobraun) [inline Cargo.toml:19]: Why `0.28.1` specifically, when `0.28.2` [is the latest version](https://crates.io/crates/quick-xml)?

**Review by hannobraun — APPROVED** [review]
Thank you, @JayceFayne, looks good!

I left one rather nit-picky comment, but it doesn't make much of a difference. Merging anyway :smile:

File: Cargo.toml (hunk 1 above)
Comment (JayceFayne) [reply Cargo.toml:19]: Weird, for some reason VS Code told me the last one is `0.28.1` D:
image

Comment (hannobraun) [reply Cargo.toml:19]: Oh well :smile:

**Pull request merged**